Basic FE project template/index.html

```diff
@@ -12,6 +12,16 @@
       rel="stylesheet"
     />
 
+    <script
```

Comment:
this does not look correct, why import 2 scripts that look the same? Why is the text "Basic usage" copied there?